pwndbg/wrappers/readelf.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import pwndbg.wrappers
+
+cmd_name = "readelf"
+
+@pwndbg.wrappers.OnlyWithCommand(cmd_name)
+def get_jmpslots():
+    local_path = pwndbg.file.get_file(pwndbg.proc.exe)
+    cmd = [get_jmpslots.cmd_path, "-r", local_path]
+    readelf_out = pwndbg.wrappers.call_cmd(cmd)
+
+    return '\n'.join(filter(_extract_jumps, readelf_out.splitlines()))
```

Comment:
We might also want to make `_extract_jumps` return a `pwndbg.elf.Relocs` or other, more meaningful named object in the future.

Just saying, we probably don't need it right now.